src/index.ts

```diff
@@ -416,24 +416,37 @@ const ruleFunction: StylelintPlugin<PrimaryOptions, SecondaryOptions> =
 
           // support auto fixing
           if (context.fix && !disableFix && autoFixFuncNormalized) {
-            const fixedValue = autoFixFuncNormalized(
-              node,
-              {
-                validVar,
-                validFunc,
-                validKeyword,
-                validValue,
-                longhandProp,
-                longhandValue,
-              },
-              root,
-              config
-            );
-
-            // apply fixed value if returned
-            if (fixedValue) {
-              // eslint-disable-next-line no-param-reassign
-              node.value = fixedValue;
+            try {
+              const fixedValue = autoFixFuncNormalized(
+                node,
+                {
+                  validVar,
+                  validFunc,
+                  validKeyword,
+                  validValue,
+                  longhandProp,
+                  longhandValue,
+                },
+                root,
+                config
+              );
+
+              // apply fixed value if returned
+              if (fixedValue) {
+                // eslint-disable-next-line no-param-reassign
+                node.value = fixedValue;
+              }
+            } catch (err: unknown) {
+              if (typeof err === 'string') {
+                utils.report({
+                  ruleName,
+                  result,
+                  node,
+                  line: node.source!.start.line,
+                  // column: start!.column + nodeProp.length + raws.between!.length,
+                  message: err,
```

Comment:
@devTeaa 
Stylelint's `utils` comes with different tools.
Another one helps you to create messages.
Please have a look at the documentation of it
https://stylelint.io/developer-guide/plugins/#stylelintutilsrulemessages

And here are examples of how they are used in this plugin:
- https://github.com/AndyOGo/stylelint-declaration-strict-value/blob/master/src/index.ts#L25-L27
- https://github.com/AndyOGo/stylelint-declaration-strict-value/blob/master/src/lib/validation.ts#L187-L227
- https://github.com/AndyOGo/stylelint-declaration-strict-value/blob/master/src/index.ts#L443-L450

Reply:
I actually used the one one the else condition under it, am I supposed to use something else?